internal/stack/_static/docker-compose-stack.yml.tmpl

```
@@ -170,11 +170,12 @@ services:
       interval: 60s
       timeout: 50s
       retries: 5
-    command: bash -c 'if [[ ! $(bin/logstash-plugin list) == *"logstash-filter-elastic_integration"* ]]; then echo "Missing plugin logstash-filter-elastic_integration, installing now" && bin/logstash-plugin install logstash-filter-elastic_integration; fi && bin/logstash -f /usr/share/logstash/pipeline/logstash.conf'
+    command: bash -c 'chmod +x /usr/share/logstash/startup.sh && /usr/share/logstash/startup.sh'
```

Could we directly run the script?
```suggestion
    command: bash /usr/share/logstash/startup.sh
```